tools/src/main/java/org/apache/rocketmq/tools/command/offset/ResetOffsetByTimeOldCommand.java

```diff
@@ -119,7 +123,7 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook) t
                 }
 
                 defaultMQAdminExt.start();
-                resetOffset(defaultMQAdminExt, consumerGroup, topic, timestamp, force, timeStampStr);
+                resetOffset(defaultMQAdminExt, consumerGroup, topic, null, timestamp, force, timeStampStr);
```

Comment:
null? maybe mistake. I thought u should convey paramater queue, but u not get queue from commandLine